walkthroughs/week-1-web-development/examples/topic_page/src/main/webapp/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Topic Page</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+      <h1 class="center"> Topic Title</h1>
+      <p class="white">Subtitle</p>
+      <p class="rectangle">Heres is the description about the subtitle</p>
+
```

Comment:
This should be done in the /topics/ folder with the html file named as the topic's id, since having multiple index.html files will cause merge conflict (homepage is index)